modules/apps/object/object-rest-impl/src/main/java/com/liferay/object/rest/internal/dto/v1_0/converter/ObjectEntryDTOConverter.java

```diff
@@ -679,13 +701,13 @@ private ExtendedEntity _toExtendedEntity(
 		return ExtendedEntity.extend(dto, nestedFieldsRelatedProperties, null);
 	}
 
-	private Map<String, Object> _toProperties(
+	private Map<String, UnsafeSupplier<Object, Exception>> _toProperties(
```

Comment:
It would be nice to refactor this method and split the smaller chunks in separate methods.